packages/backend/package.json

```diff
@@ -1,14 +1,34 @@
 {
   "name": "backend",
   "scripts": {
-    "build": "tsc --build tsconfig.json"
+    "build": "tsc --build tsconfig.json",
+    "knex": "./node_modules/.bin/knex --knexfile src/database/knexfile.js"
```

Comment:
if the knex file is moved as per my previous comment of this review, I think we can then just have this script as:
"kenx": knex